rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -210,6 +214,9 @@ static bool encl_build(struct sgx_secs *secs, void *bin, unsigned long bin_size,
 		return false;
 	}
 
+	if (!(sigstruct->body.attributes & SGX_ATTR_DEBUG))
+		enclave_debug = false;
```

Comment:
This chunk of code is not necessary. enclave_debug is only controlled by the user input.